### ISSUE TITLE
Self+Shink server crash

### ISSUE THREAD

**Issue by CrowAU** [Mithion/ArsMagica2#1407]
The self+shrink spell causes a ticking entity issue

Crashlog: http://puu.sh/n1lhb/94911f9a9b.txt

**Comment by jjtParadox** [comment]
This ventures into the realm of ASM, a realm where many end up lost...

This probably needs to be tested with just AM2 to see if that causes the issue, and then figure out if there's some kind of issue with mod interaction.

**Comment by ChilledDayz** [comment]
I'm currently playing just am2, I can test this later for you if it helps.

**Comment by jjtParadox** [comment]
Yeah that would help. Life's getting a little busy so testing and coding stuff is going to be a little hard.

**Comment by ChilledDayz** [comment]
I can't help with coding but I can help test issues :-)
